Project.cmake

```diff
@@ -36,6 +37,8 @@ if(WIN32)
 
 	# Install binaries in the root of the install directory
 	set(INSTALL_BIN_DIR "/" CACHE PATH "Where to place binaries in the instalation")
+elseif(APPLE)
+	set(INSTALL_BIN_DIR "~/Applications" CACHE PATH "Where to place binaries in the instalation")
```

Comment:
Shouldn't this be `Contents/MacOS/`? See [this](https://developer.apple.com/library/mac/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW19).

I usually interpret the install directory as "what goes inside of generated packages".